parent/pom.xml

```diff
@@ -0,0 +1,516 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>net.java.dev.jna</groupId>
+    <artifactId>parent</artifactId>
+    <version>4.3.0-SNAPHSOT</version>
+    <packaging>pom</packaging>
+    <name>net.java.dev.jna:parent</name>
+    <url>https://github.com/java-native-access/jna</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <!-- ### TODO find some way to import build.version.properties ### -->
+
+          <!-- expected JVM versions -->
+        <jvm.major.version>1</jvm.major.version>
+        <jvm.minor.version>5</jvm.minor.version>
+        <jvm.release.version>0</jvm.release.version>
+
+        <javac.source>${jvm.major.version}.${jvm.minor.version}</javac.source>
+        <project.build.java.source>${javac.source}</project.build.java.source>
+        <maven.compiler.source>${javac.source}</maven.compiler.source>
+        <ant.build.javac.source>${javac.source}</ant.build.javac.source>
+
+        <javac.target>${javac.source}</javac.target>
+        <project.build.java.target>${javac.target}</project.build.java.target>
+        <maven.compiler.target>${javac.target}</maven.compiler.target>
+        <ant.build.javac.target>${javac.target}</ant.build.javac.target>
+
+        <min.required.maven.version>3.2</min.required.maven.version>
+        <surefire.plugin.version>2.19</surefire.plugin.version>
+            <groovy.major.version>2</groovy.major.version>
+            <groovy.minor.version>4</groovy.minor.version>
+            <groovy.release.version>5</groovy.release.version>
+            <groovy.compliance.level>2.0</groovy.compliance.level>
+        <groovy.version>${groovy.major.version}.${groovy.minor.version}.${groovy.release.version}</groovy.version>
+        <gmaven.plugin.version>1.5</gmaven.plugin.version>
+
+        <ant.version>1.9.6</ant.version>
+        <xmltask.version>1.16</xmltask.version>       
+        <ant-contrib.version>1.0b3</ant-contrib.version>
+        <junit.version>4.12</junit.version>
+    </properties>
+
+    <prerequisites>
+        <maven>${min.required.maven.version}</maven>
+    </prerequisites>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
```

Comment:
I don't think these ant dependencies are needed because they are not used as dependencies. Further down, they are declared as deps of a plugin in <plugin-mgt> section, which should suffice.